tests/py/test_boost_python.py

```diff
@@ -66,3 +66,32 @@ def set_xsd_optional(optional):
 	assert optional == XSDType.STRING, "optional is not STRING"
 	optional = XSDType.DOUBLE
 	assert optional == XSDType.DOUBLE, "optional is not DOUBLE"
+
+
+def query_knowledge_base(optional):
+	# Initialize the knowledge base
+	InitKnowledgeBase()
```

Comment:
should not be done in the test. for unit tests it is done here **https://github.com/knowrob/knowrob/blob/dev/tests/gtests.cpp#L6**

Reply:
I did comment this call out for now